go/dpagg/sum.go

```diff
@@ -249,6 +251,28 @@ func (bs *BoundedSumInt64) ThresholdedResult(thresholdDelta float64) *int64 {
 	return &result
 }
 
+// ComputeConfidenceInterval computes a confidence interval with integer bounds that contains the true sum with
+// a probability greater than or equal to 1 - alpha using the noised sum computed by Result(). Note Result()
+// needs to be called before ComputeConfidenceInterval, otherwise this will return an error.
+func (bs *BoundedSumInt64) ComputeConfidenceInterval(alpha float64) (noise.ConfidenceInterval, error) {
+	if !bs.resultReturned {
+		return noise.ConfidenceInterval{}, fmt.Errorf("You need to call Result() before calling ComputeConfidenceInterval()")
+	}
+	confInt, err := bs.noise.ComputeConfidenceIntervalInt64(bs.noisedSum, bs.l0Sensitivity, bs.lInfSensitivity, bs.epsilon, bs.delta, alpha)
+	if err != nil {
+		return noise.ConfidenceInterval{}, err
+	}
+	// If lower and upper bounds are non-negative, trim any negative interval.
```

Comment:
There is single interval here right? (i.e. (lower, upper) constitute a single interval)

So it might be a good idea to replace `trim any negative interval` with `trim the negative part of the interval`, or `clamp the interval to be non-negative`.

(Same comment below)